src/test/java/io/github/oliviercailloux/collaborative_exams/controller/ChangeQuestionLanguaugeTest.java

```diff
@@ -22,8 +26,14 @@
 
 	@Deployment(testable = false)
 	public static WebArchive createDeployment() {
-		final WebArchive war = ShrinkWrap.create(WebArchive.class, "collaborative-exams-0.0.1.war")
-				.addPackage(ChangeQuestionLanguaugeTest.class.getPackage());
+		final File[] libs = Maven.resolver().loadPomFromFile("pom.xml").importRuntimeDependencies().resolve()
+				.withTransitivity().asFile();
+		
+		final WebArchive war = ShrinkWrap.create(WebArchive.class, "examcollab.war")
```

Comment:
Pourquoi cette (quasi-)quadruple répétition plutôt qu’une seule méthode, appelée quatre fois ? Documenter si la différence de cette invocation (avec addAsLibraries) est importante.

Reply:
Il ne m'est actuellement pas possible d'effectuer des tests et ce depuis une dizaine d'heures. Les jobs Travis restent bloqués sur "queued".

L'invocation me semble aussi inutile puisque les dépendances sont déjà renseignées dans le pom.xml en suivant ce tutoriel : https://deltaspike.apache.org/documentation/configure.html

Mais il semble que cela soit nécessaire comme vous pouvez le voir dans les posts suivants : 
https://developer.jboss.org/thread/237174
http://abulletproofidea.com/questions/50927/test-cdi-avec-arquillian
https://developer.jboss.org/thread/237358
https://stackoverflow.com/questions/33169361/arquillian-configuration-deploymentexception-with-windowbeanholder
http://apache-deltaspike-incubator-discussions.2316169.n4.nabble.com/Wled-2-1-2-Could-not-find-beans-for-Type-class-org-apache-deltaspike-core-impl-scope-window-WindowBe-td4656801.html
https://users.openejb.apache.narkive.com/5cYxHyxG/applicationcomposer-and-deltaspike-1-0